lib/karafka/helpers/inflector.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Karafka
+  module Helpers
+    module Inflector
+      ENGINE = Dry::Inflector.new
+
+      def self.underscore(element)
+        ENGINE.underscore(element.to_s).sub('/', '_')
```

Comment:
I think this should be `gsub` instead of `sub`. Currently:

```
2.5.1 :003 > Karafka::Helpers::Inflector.underscore('ModuleA::ModuleB::ModuleC::MyConsumer')
 => "module_a_module_b/module_c/my_consumer"
```